docs/docusaurus/docs/guides/expectations/creating_custom_expectations/add_custom_parameters.md

```diff
@@ -0,0 +1,91 @@
+---
+sidebar_label: 'Add custom parameters to Custom Expectations'
+title: 'Add custom parameters to Custom Expectations'
+id: add_custom_parameters
+description: Add custom parameters to Custom Expectations.
+---
+
+Using custom parameters in your Custom Expectations can help you create powerful, business-specific validations and help you optimize your Great Expectations (GX) workflows. Custom parameters should be implemented when you have data dependencies that can’t or shouldn’t be hardcoded. The method of implementing custom parameters is specific to the Custom Expectation class being extended.
+
+## Implementation
+
+All Custom Expectations include a `success_keys` attribute that is a tuple of strings. The tuple items are the names of the parameters which are available during the evaluation of your Custom Expectation. For example, the `success_keys` attribute for the ColumnPairMapExpectation appears similar to the following example:
+
+```python
+success_keys = (
+   "column_A",
+   "column_B",
+   "mostly",
+)
+```
+The tuple in the example includes the two columns being evaluated and the `mostly` parameter. These parameters are passed as a part of your ExpectationConfiguration and are available to the Expectation `_validate` method.
+
+## Use case
+
+In this use case, a `condition_value_keys` tuple and a `condition_domain_keys` tuple are added to ColumnPairMapMetrics, MulticolumnMapMetrics, and ColumnMapMetrics. The `condition_value_keys` tuple supplies the arguments needed to compute your Metric, and the `condition_domain_keys` tuple defines the domain on which the Metric operates.
+
+This is how the tuples appear in the MulticolumnMapMetrics Metric:
+
+```python
+condition_domain_keys = (
+   "batch_id",       
+   "table",       
+   "column_list",       
+   "row_condition",       
+   "condition_parser",       
+   "ignore_row_if",   
+)   
+condition_value_keys = ("sum_total",)
+
+ColumnAggregateMetrics, TableMetrics, and QueryMetrics can similarly define a value_keys tuple or a domain_keys tuple:
+
+value_keys = ("column",)
+domain_keys = ("query",)
+```
```

Comment:
Some ambiguity here -- condition_domain_keys & condition_value_keys are utilized in all *MapMetrics (ColumnMap, ColumnPairMap, and MultiColumnMap)

ColumnAggregate, Batch (formerly Table), and Query all use value_keys & domain_keys instead.